packages/prettier-plugin-liquid/src/printer/preprocess/augment-with-css-properties.ts

```diff
@@ -177,7 +180,12 @@ function getNodeCssStyleWhiteSpace(node: AugmentedNode<WithSiblings>): string {
       return 'pre';
 
     case NodeTypes.LiquidTag:
-      return CSS_WHITE_SPACE_LIQUID_TAGS[node.name] || CSS_WHITE_SPACE_DEFAULT;
+      switch (options.captureWhitespace) {
+        case 'strict':
+          return (node.name === 'capture') ? 'pre' : 'normal';
+        case 'ignore':
+          return (node.name === 'capture') ? 'normal' : 'pre';
+      }
```

Comment:
Here I think that—since this is an option for `capture` tags only—I'd do a switch on node name first, and then on the option. Also note that we also want `pre` for `raw` tags so I don't think the switch on options.captureWhitespace works well here. We should defer to CSS_WHITE_SPACE_LIQUID_TAGS[node.name] instead. 
```suggestion
      switch (node.name) {
        case NamedTags.capture: {
          switch (options.captureWhitespace) {
            case 'strict':
              return 'pre';
            case 'ignore':
              return 'normal';
            default: {
              throw assertNever(options.captureWhitespace);
            }
          }
        }

        default: {
          return CSS_WHITE_SPACE_LIQUID_TAGS[node.name] || CSS_WHITE_SPACE_DEFAULT;
        }
```